qtp_target_gene/validate.py

```diff
@@ -321,7 +321,7 @@ def _check_files(run_prefixes, read_files, rev_count, files):
             except OSError:
                 fp_size = 0
             # 62 is the size of a gzip empty files that we generate
-            if fp_size <= 62:
+            if fp_size <= 100:
```

Comment:
What about for checking the contents of the file instead of the size (since this could potentially change again). Oddly enough I ran into this last week, and found this [post on SO](https://stackoverflow.com/a/37878550/379593).

Reply:
Thought about it and I even started implementing it and realized that the number of changes will make it not worth it ...